covid/question_covid.py

```diff
@@ -67,9 +67,10 @@ def predict(self, question):
             with open(paper_path) as json_file:
                 article_data = json.load(json_file)
                 text = ' '.join([d['text'] for d in article_data['body_text']])
-
-            for text_i in range(0, len(text), 512):
-                subtext = text[text_i:text_i+512]
+            sentences = text.split('.')
```

Comment:
quite simple, this might create problematic splits down the line but i think it is fair enough for now. in the future we could experiment with scispacy that should be better in splitting sentences on that domain.